pom.xml

```diff
@@ -23,38 +23,43 @@ under the License.
     <parent>
         <groupId>org.apache.netbeans</groupId>
         <artifactId>netbeans-parent</artifactId>
-        <version>3</version>
+        <version>4</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.apache.netbeans.archetypes</groupId>
     <artifactId>netbeans-platform-app-archetype</artifactId>
     <version>1.24-SNAPSHOT</version>
     <packaging>maven-archetype</packaging>
-    <name>Apache Netbeans Maven Utilties - NetBeans Platform Application Archetype</name>
+    <name>Apache Netbeans Maven Utilities - NetBeans Platform Application Archetype</name>
```

Comment:
btw can we make this shorter? This doesn't fit into the outline view of NB.

Reply:
maybe Apache Netbeans Platform Application Maven Archetype ?

Reply:
this is still a sentence and not a name :)

Reply:
lets call it "NetBeans Platform Application Archetype"? rest is in the description

cc @neilcsmith-net

Reply:
I think we "need" the Apache associated with NetBeans

Reply:
changed it to "Apache NetBeans Platform Application Archetype" which is half of the original length.